Restrict colors to a subset of "brand colors"

ryelle opened the pull request:
This PR was created with [an automated command](https://github.com/ryelle/wp-core-color-list/tree/main/postcss) to reduce the colors used in CSS to a fixed set. See the [color mappings visually here.](https://inspiring-stonebraker-7fc5ed.netlify.app/) By using an automated tool, we can keep the CSS changes in sync with trunk, without excessive manual effort.

This likely still needs manual editing to update places where the colors don't work anymore (for example, the hover background on the admin menu has disappeared, as has the light blue background on active plugins). Oranges have also become reds, since the [color palette used](https://i2.wp.com/make.wordpress.org/design/files/2019/11/image-21-scaled.png?ssl=1) does not have orange.

| Current (trunk) | New Colors |
|----------------|-------------|
| ![trunk-dashboard](https://user-images.githubusercontent.com/541093/91874338-1a398880-ec48-11ea-9826-489ea17a3a61.png) | ![new-colors-dashboard](https://user-images.githubusercontent.com/541093/91874326-1574d480-ec48-11ea-97dc-3f7d9bf1fd08.png) |
| ![trunk-menu](https://user-images.githubusercontent.com/541093/91874345-1c034c00-ec48-11ea-9d87-93e51ed9ef47.png) | ![new-colors-menu](https://user-images.githubusercontent.com/541093/91874333-19a0f200-ec48-11ea-825d-df876c78fe65.png) |
| ![trunk-media-library](https://user-images.githubusercontent.com/541093/91874341-1ad21f00-ec48-11ea-9592-431d989bae47.png) | ![new-colors-media-library](https://user-images.githubusercontent.com/541093/91874328-16a60180-ec48-11ea-8b4e-4dcc96231234.png) |
| ![trunk-plugins](https://user-images.githubusercontent.com/541093/91874348-1c034c00-ec48-11ea-9b97-8fdc5a7850fd.png) | ![new-colors-plugins](https://user-images.githubusercontent.com/541093/91874336-1a398880-ec48-11ea-8929-8ec0426e1130.png) |

ryelle committed:
Restrict colors to a subset of "brand colors"

laras126 commented:
This is so cool!!! 🥳

Definitely will be a need for some contrast checks – the menu link blue looks like potentially a problem at first glance. This seems like a good starting point to have designers to step in to review, or at least have an initial look.

Also – it's easier to visualize how the custom properties will come into play now :-D

erikjoling commented:
Just dropping in here. I thought I have seen some color development in the Gutenberg repository. Global Styles seems to have overlap with your efforts. I can't find the right issue, but you might check out https://github.com/WordPress/gutenberg/issues/19611 and @itsJonQ.

laras126 commented:
@erikjoling - maybe this? https://github.com/ItsJonQ/g2/issues/19

ryelle committed:
Update green in "copied" message

ryelle committed:
Update active plugins to use lightest blue

See https://github.com/ryelle/wp-core-color-list/issues/7

SergeyBiryukov committed:
External Libraries: Upgrade PHPMailer to version 6.2.0.

For a full list of changes in this update, see the PHPMailer GitHub:
https://github.com/PHPMailer/PHPMailer/compare/v6.1.8...v6.2.0

Props ayeshrajans, jrf.
Fixes #51874.

git-svn-id: https://develop.svn.wordpress.org/trunk@49713 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
Site Health, App Passwords: Ensure REST API responses are properly translated.

The REST API requests in Site Health and App Passwords now include `_locale=user` in the request URL to ensure the user's locale is used instead of the site locale. Additionally, the `apiRequest` library now sends a JSON `Accept` header which is required by `determine_locale()` to respect the `_locale` query parameter.

The Site Health REST API controllers now manually load the default admin textdomain if not `is_admin()`. This allows for the Site Health tests to be translated even though the translations are part of the administration project and the REST API is not.

Props oglekler, kebbet, Clorith, TimothyBlynJacobs, ocean90, SergeyBiryukov, adamsilverstein.
Fixes #51871.


git-svn-id: https://develop.svn.wordpress.org/trunk@49716 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Editor: Update packages for WordPress 5.6 RC2.

The following package versions were changed:

@wordpress/block-directory: 1.17.4 -> 1.17.5
@wordpress/block-editor: 5.1.3 -> 5.1.4
@wordpress/block-library: 2.26.4 -> 2.26.5
@wordpress/blocks: 6.24.1 -> 6.24.2
@wordpress/components: 11.1.1 -> 11.1.2
@wordpress/core-data: 2.24.1 -> 2.24.2
@wordpress/edit-post: 3.25.4 -> 3.25.5
@wordpress/editor: 9.24.3 -> 9.24.4
@wordpress/format-library: 1.25.3 -> 1.25.4
@wordpress/list-reusable-blocks: 1.24.1 -> 1.24.2
@wordpress/nux: 3.23.1 -> 3.23.2
@wordpress/reusable-blocks: 1.0.3 -> 1.0.4
@wordpress/server-side-render: 1.19.1 -> 1.19.2

See https://github.com/WordPress/gutenberg/pull/27366 for the list of fixed bugs.

Props isabel_brison.
Fixes #51902.

git-svn-id: https://develop.svn.wordpress.org/trunk@49718 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Media: Revert [49567].

This addresses a regression with the pagination section in Media Library no longer taking additional query filtering into account.

Props iCaleb, trepmal, peterwilsoncc.
See #39968.

git-svn-id: https://develop.svn.wordpress.org/trunk@49720 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Editor: Don't unnecessarily split a translatable string in block templates.

As a best practice, strings available for translation should contain entire sentences whenever possible.

Splitting a sentence in two parts and putting them back together after translation should be avoided, as the word order in other languages can be different from English.

Props tobifjellner, kebbet, audrasjb, mukesh27, hellofromTonya, azaozz, SergeyBiryukov.
Fixes #51893.

git-svn-id: https://develop.svn.wordpress.org/trunk@49722 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Administration: Make sure row actions for recent comments in Activity dashboard widget stay visible when a single row gets focus.

Follow-up to [49142].

Props sabernhardt, kebbet, hareesh-pillai, pbiron.
Fixes #51886. See #51516, #49715.

git-svn-id: https://develop.svn.wordpress.org/trunk@49725 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Twenty Twenty-One: Sync the latest changes for 5.6 RC2.

This will be the final sync from GitHub before placing that repository into read-only mode. All further changes should now flow entirely through Trac.

For a full list of changes since [49633], see https://github.com/WordPress/twentytwentyone/compare/1d5a895...53acd9b.

Props poena, luminuu, kjellr, ryelle, allancole, melchoyce, felipeelia, aljullu, kebbet, chaton666, Clorith, mkaz, ingereck, paaljoachim.
See #51526.

git-svn-id: https://develop.svn.wordpress.org/trunk@49726 602fd350-edb4-49c9-b593-d223f7449a82

johnbillion committed:
REST API: Pass the previous state of the post as a parameter to the `wp_after_insert_post` hook.

This enables, for example, the previous post status to be used by this hook without the need to first capture it on an earlier hook.

This also fixes the value of the `$fire_after_hooks` parameter in `get_default_post_to_edit()` so the `wp_after_insert_post` action correctly fires just once on the new post screen.

Props Collizo4sky, peterwilsoncc, hellofromTonya, TimothyBlynJacobs, SergeyBiryukov

Fixes #45114


git-svn-id: https://develop.svn.wordpress.org/trunk@49731 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Docs: Clarify the `$format` parameter description in `comment_time()`.

See #51800.

git-svn-id: https://develop.svn.wordpress.org/trunk@49735 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Upgrade/Install: Check `$wp_version` global for displaying "You are using a development version" message in the admin footer.

This brings some consistency with the same check in `core_upgrade_preamble()` and avoids a PHP warning if `$cur->version` is not set.

Additionally, remove the check for `$cur->url` property, unused since [8595].

Follow-up to [49708], [49709].

Props pbiron, afragen, audrasjb.
Fixes #51892.

git-svn-id: https://develop.svn.wordpress.org/trunk@49736 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Editor: Update components package for WordPress 5.6 RC3.

@wordpress/components: 11.1.2 -> 11.1.3

This fixes an accessibility regression with VoiceOver not announcing the input label or an existing selection in the parent page dropdown under "Page Attributes" in the sidebar.

See https://github.com/WordPress/gutenberg/pull/27431 for more details.

Props isabel_brison, sarahricker, alexstine, kevin940726, talldanwp.
Fixes #51923.

git-svn-id: https://develop.svn.wordpress.org/trunk@49737 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
App Passwords: Return true when rate limiting a password's last used time.

Previously we returned a `WP_Error` instance saying that the password was not found which is inaccurate.

Props dlh.
Fixes #51922.


git-svn-id: https://develop.svn.wordpress.org/trunk@49739 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Upgrade/Install: Update sodium_compat to v1.14.0.

This includes improved PHP 8 support and more inclusive language.

A full list of changes in this update can be found on GitHub:
https://github.com/paragonie/sodium_compat/compare/v1.13.0...v1.14.0

Follow-up to [48121], [49056], [49057].

Props jrf.
Fixes #51925.

git-svn-id: https://develop.svn.wordpress.org/trunk@49741 602fd350-edb4-49c9-b593-d223f7449a82

iandunn committed:
Multisite: Cache absolute `dirsize` paths to avoid PHP 8 fatal.

r49212 greatly improved the performance of `get_dirsize()`, but also changed the structure of the data stored in the `dirsize_cache` transient. It stored relative paths instead of absolute ones, and also removed the unnecessary `size` array.

That difference in data structures led to a fatal error in the following environment:

* PHP 8
* Multisite
* A custom `WP_CONTENT_DIR` which is not a child of WP's `ABSPATH` folder (e.g., [https://roots.io/bedrock/ Bedrock])
* The `upload_space_check_disabled` option set to `0`

After upgrading to WP 5.6, the `dirsize_cache` transient still had data in the old format. When `wp-admin.php/index.php` was visited, `get_space_used()` received an `array` instead of an `int`, and tried to divide it by another `int`. PHP 7 would silently cast the arguments to match data types, but [https://wiki.php.net/rfc/arithmetic_operator_type_checks PHP 8 throws a fatal error]: 

`Uncaught TypeError: Unsupported operand types: array / int`

`recurse_dirsize()` was using `ABSPATH` to convert the absolute paths to relative ones, but some upload locations are not located under `ABSPATH`. In those cases, `$directory` and `$cache_path` were identical, and that triggered the early return of the old `array`, instead of the expected `int`. 

In order to avoid that, this commit restores the absolute paths, but without the `size` array. It also adds a type check when returning cached values. Using absolute paths without `size` has the result of overwriting the old data, so that it matches the new format. The type check and upgrade routine are additional safety measures.

Props peterwilsoncc, janthiel, helen, hellofromtonya, francina, pbiron.
Fixes #51913. See #19879.



git-svn-id: https://develop.svn.wordpress.org/trunk@49744 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
App Passwords: Ensure the Created At and Last Used dates are properly translated.

The `date_i18n` function is now used when formatting the dates in PHP instead of `gmdate` which doesn't handle localization properly.

Additionally, we now use a translation to get the date format to use instead of pulling from the `date_format` option which is only supposed to affect the front-end.

Lastly, when passing the date format to the Backbone JS template, we now use `wp_json_encode()` to format the value for JavaScript. This ensures that backslashes are properly preserved which are used by some locales to escape date formatting control characters.

Props pedromendonca, TimothyBlynJacobs, ocean90, hellofromtonya, SergeyBiryukov, antpb.
Fixes #51918.
See [35811].


git-svn-id: https://develop.svn.wordpress.org/trunk@49746 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Database: Check the correct value for displaying the "Successfully repaired table" or "Failed to repair" messages on Database Repair screen.

Additionally, rename the variables holding the results of `ANALYZE TABLE` and `OPTIMIZE TABLE` requests, for clarity.

Props mehrshaddarzi.
Fixes #51932.

git-svn-id: https://develop.svn.wordpress.org/trunk@49750 602fd350-edb4-49c9-b593-d223f7449a82

iandunn committed:
Media: Return `WP_Error` when cropping with bad input to avoid fatal.

This avoids an error on PHP 8 caused by calling `wp_imagecreatetruecolor()` with inputs that aren't numeric, or are less than 0.

Props hellofromtonya, Boniu91, metalandcoffee, SergeyBiryukov.
Fixes #51937.


git-svn-id: https://develop.svn.wordpress.org/trunk@49751 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
App Passwords: Prevent conflicts when Basic Auth is already used by the site.

Application Passwords uses Basic Authentication to transfer authentication details. If the site is already using Basic Auth, for instance to implement a private staging environment, then the REST API will treat this as an authentication attempt and would end up generating an error for any REST API request.

Now, Application Password authentication will only be attempted if Application Passwords is in use by a site. This is flagged by setting an option whenever an Application Password is created. An upgrade routine is added to set this option if any App Passwords already exist.

Lastly, creating an Application Password will be prevented if the site appears to already be using Basic Authentication.

Props chexwarrior, georgestephanis, adamsilverstein, helen, Clorith, marybaum, TimothyBlynJacobs.
Fixes #51939.



git-svn-id: https://develop.svn.wordpress.org/trunk@49752 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Tests: Correct `@ticket` references in some test files.

See #51802.

git-svn-id: https://develop.svn.wordpress.org/trunk@49757 602fd350-edb4-49c9-b593-d223f7449a82

peterwilsoncc committed:
Docs, Tests: Correctly capitalize JavaScript.

Correct Javascript to JavaScript within Wordpress core ;)

See #51800, #51802.



git-svn-id: https://develop.svn.wordpress.org/trunk@49758 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Fix the nesting of the `main` element.

The `main` element must not appear as a descendant of the `section` element. Correct markup is the first requirement to make user agents and assistive technologies work properly.

Changes the `<section>` element that was wrapping the `<main>` element to a `<div>`.

Follow-up to [45942] for Twenty Nineteen.

Props albertomake, poena.
Fixes #51944.
}}}

git-svn-id: https://develop.svn.wordpress.org/trunk@49759 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Use consistent HTML comments after closing HTML tags.

Follow-up to [45209] for other bundled themes.

Props arcangelini, audrasjb.
Fixes #51950.

git-svn-id: https://develop.svn.wordpress.org/trunk@49761 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Use `esc_url()` for the WordPress.org link in `footer.php`.

Props mukesh27, poena.
Fixes #51954.

git-svn-id: https://develop.svn.wordpress.org/trunk@49762 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Bundled Themes: Bump theme versions ahead of WordPress 5.6.

The following theme versions will be released in coordination with WordPress 5.6:

* Twenty Ten: 3.2
* Twenty Eleven: 3.6
* Twenty Twelve: 3.3
* Twenty Thirteen: 3.2
* Twenty Fourteen: 3.0
* Twenty Fifteen: 2.8
* Twenty Sixteen: 2.3
* Twenty Seventeen: 2.5
* Twenty Nineteen: 1.8
* Twenty Twenty: 1.6
* Twenty Twenty-One: 1.0

Props peterwilsoncc, hareesh-pillai, dkarfa, sabernhardt, poena, aristath, SergeyBiryukov.
Fixes #51919.

git-svn-id: https://develop.svn.wordpress.org/trunk@49763 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
App Passwords: Store the "in use" option in the main network options.

Whether App Passwords are being used is a global featurel, not a per-network feature. This fixes issues on Multi Network installs if App Passwords are used on a different network from where they were created.

Props spacedmonkey.
Fixes #51939.
See [49752].


git-svn-id: https://develop.svn.wordpress.org/trunk@49764 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Docs: Fix typo in references to `WP_Error` in some DocBlocks.

See #51800.

git-svn-id: https://develop.svn.wordpress.org/trunk@49769 602fd350-edb4-49c9-b593-d223f7449a82

ryelle committed:
Help/About: Use CDN URLs for About page images.

See #51415.



git-svn-id: https://develop.svn.wordpress.org/trunk@49770 602fd350-edb4-49c9-b593-d223f7449a82

TimothyBJacobs committed:
App Passwords: Ensure redirection description is shown on mobile devices.

Props #core.
Fixes #51970.


git-svn-id: https://develop.svn.wordpress.org/trunk@49772 602fd350-edb4-49c9-b593-d223f7449a82

helen committed:
Update `$_old_files` for 5.6.

See #39943.


git-svn-id: https://develop.svn.wordpress.org/trunk@49774 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Remove the Appveyor configuration file.

Appveyor was added to Core in [44594] to ensure that NPM and the WordPress build tools continue to work correctly on Windows systems as changes are made. Using an additional service for this was required at the time, TravisCI did not support testing on Windows.

[49162] introduced a GitHub Action workflow that performed the same testing. Since all automated testing is moving to GitHub Actions, using Appveyor is no longer necessary, and it’s preferable to have all automated testing in one location as much as possible.

Props ayeshrajans.
Fixes #51968.

git-svn-id: https://develop.svn.wordpress.org/trunk@49779 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Update `package-lock.json` file with several package updates.

These updates result from running `npm install`.

Also, update the version of `grunt-patch-wordpress` in the lock file from `3.0.0` to `3.0.1`.

See #51801.

git-svn-id: https://develop.svn.wordpress.org/trunk@49780 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Disable GitHub Action workflow runs triggered on `push` for forks and mirrors.

In the current state, the workflows run regardless of the repository context. This results many needless workflow runs that waste resources.

Workflow runs for private repositories are not free (accounts have a finite allotment of minutes for private repositories). This becomes problematic in private repositories that also mirror the WordPress develop repository, as any workflow runs will draw from a user’s or organization’s allotted action minutes.

Without blanket disabling the workflow manually for all event triggers, or modifying the workflow files in the forked/mirrored repository, there is no way to tune when the workflows run.

This change introduces a conditional statement into all GitHub Action workflows that prevents them from running on forked/mirrored repositories that are not `wordpress-develop`, except when a pull request is being submitted to that repository.

The exception to this is the Welcome workflow that posts a helpful message to first time contributors to `wordpress-develop`. This message is specific to this repository, so should only run when a PR is submitted to that repository.

See #50401.

git-svn-id: https://develop.svn.wordpress.org/trunk@49781 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Use NodeJS 14 in GitHub Action workflows.

14.x is the current LTS release, and is what `trunk` currently runs.

See #50401.

git-svn-id: https://develop.svn.wordpress.org/trunk@49782 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Enable reporting of results to WordPress.org.

This configures reporting of the test results to the WordPress.org Host Test Results in the new GitHub Actions workflow for PHPUnit testing.

See https://make.wordpress.org/hosting/test-results/

Props mikeschroder, dd32.
See #50401.

git-svn-id: https://develop.svn.wordpress.org/trunk@49783 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Inline documentation updates for the PHPUnit workflow.

See #50401.

git-svn-id: https://develop.svn.wordpress.org/trunk@49784 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Docs: Fix typo in some DocBlocks.

See #51800.

git-svn-id: https://develop.svn.wordpress.org/trunk@49785 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Reestablish the codebase as a Git repo before reporting test results.

The test reporter uses the hidden `.svn` or `.git` directories to determine which changeset to send to WordPress.org along with the test report.

Because the GitHub Action workflow builds and prepares WordPress in a preceding job to share with each test job, the `.git` directory is missing (it is purposefully not included to limit artifact size).

This re-establishes the directory as a checkout of the `wordpress-develop` repository so that test results can be reported correctly.

See #50401.

git-svn-id: https://develop.svn.wordpress.org/trunk@49786 602fd350-edb4-49c9-b593-d223f7449a82

johnbillion committed:
Docs: Fix some docblock issues introduced in 5.6.

See #51800


git-svn-id: https://develop.svn.wordpress.org/trunk@49787 602fd350-edb4-49c9-b593-d223f7449a82

johnbillion committed:
Upgrade/Install: Remove a duplicate description for the `upgrader_overwrote_package` action.

See #51800, #9757


git-svn-id: https://develop.svn.wordpress.org/trunk@49788 602fd350-edb4-49c9-b593-d223f7449a82

johnbillion committed:
Docs: Various docblock corrections.

See #51800


git-svn-id: https://develop.svn.wordpress.org/trunk@49789 602fd350-edb4-49c9-b593-d223f7449a82

johnbillion committed:
Docs: Various docblock improvements.

See #51800


git-svn-id: https://develop.svn.wordpress.org/trunk@49790 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Update several NPM packages.

These were updated using `npm audit fix`.

See #51801.

git-svn-id: https://develop.svn.wordpress.org/trunk@49791 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Docs: Document the usage of `$wpdb` global in `WP_Query::get_posts()` and `::set_found_posts()`.

Props jontyravi.
Fixes #52027.

git-svn-id: https://develop.svn.wordpress.org/trunk@49792 602fd350-edb4-49c9-b593-d223f7449a82

ryelle committed:
Twenty Nineteen: Add images for use in block patterns.

The block patterns were added in #51099, but the images were not included in the commit. This adds the images to the expected folder.

Props transl8or, mukesh27.
Fixes #51996.



git-svn-id: https://develop.svn.wordpress.org/trunk@49793 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Correct version in `file-header.scss`.

Fixes #52044.

git-svn-id: https://develop.svn.wordpress.org/trunk@49794 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Fix typos in some inline comments.

Props tanvirul, mukesh27.
Fixes #52042.

git-svn-id: https://develop.svn.wordpress.org/trunk@49795 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Remove PHPUnit 3.6.x specific settings from the multisite configuration.

In PHPUnit 3.6.x, `exclude` and `file` paths were required to be relative to the checkout root. This changed in PHPUnit 4 when the path requirements were changed require them to be relative to the configuration file.

Since PHPUnit 3.6.x is only used to test on PHP 5.2 in older branches and is no longer supported, these can be safely removed from the configuration file.

See #51802.

git-svn-id: https://develop.svn.wordpress.org/trunk@49796 602fd350-edb4-49c9-b593-d223f7449a82

desrosj committed:
Build/Test Tools: Allow code coverage reports to be generated when running tests as a multisite.

This adds the information needed to generate a code coverage report to the multisite ruleset.

Fixes #52034.

git-svn-id: https://develop.svn.wordpress.org/trunk@49797 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Use a more specific link for Dark Mode instructions.

Make both instances of the link translatable, for consistency.

Props engahmeds3ed, justinahinon.
Fixes #52010.

git-svn-id: https://develop.svn.wordpress.org/trunk@49798 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Correct the DocBlock for `map-deep-set()` Sass function.

Props mukesh27.
Fixes #52064.

git-svn-id: https://develop.svn.wordpress.org/trunk@49799 602fd350-edb4-49c9-b593-d223f7449a82

ryelle committed:
Twenty Twenty-One: Bring back PostCSS config.

Twenty Twenty-One uses PostCSS to convert the custom properties for Internet Explorer. Without a config file, the postcss command does nothing, causing issues in IE. This brings back the config from GitHub, which replaces all custom properties with the default values.

Props t-p, poena.
Fixes #52040.



git-svn-id: https://develop.svn.wordpress.org/trunk@49800 602fd350-edb4-49c9-b593-d223f7449a82

SergeyBiryukov committed:
Twenty Twenty-One: Fix typos in some inline comments.

Props manzurahammed, mukesh27.
Fixes #52071.

git-svn-id: https://develop.svn.wordpress.org/trunk@49801 602fd350-edb4-49c9-b593-d223f7449a82

ryelle committed:
Update unapproved comment background color
